Dockerfile

```diff
@@ -5,22 +5,46 @@ ADD super_netperf   /sbin/
 ADD percpu_netperf  /sbin/
 ADD latency_netperf /sbin/
 
-RUN											   \
-	apk add --update curl wrk iperf3 git build-base texinfo bash iproute2 automake     \
-			 libtool intltool autoconf					&& \
-	git clone https://github.com/HewlettPackard/netperf.git				&& \
-	cd netperf/									&& \
-	./autogen.sh									&& \
-	chmod a+x configure								&& \
-	./configure --prefix=/usr							&& \
-	make										&& \
-	make install									&& \
-	cd ..										&& \
-	rm -rf netperf									&& \
-	rm -f /usr/share/info/netperf.info						&& \
-	strip -s /usr/bin/netperf /usr/bin/netserver					&& \
-	apk del build-base texinfo git automake libtool intltool autoconf		&& \
-	rm -rf /var/cache/apk/*
+RUN	apk add --update \
+	curl \
+	wrk \
+	git \
+	build-base \
+	texinfo \
+	bash \
+	iproute2 \
+	automake \
+	libtool \
+	intltool \
+	autoconf \
+	lksctp-tools-dev \
+	linux-headers
+
+# The CFLAGS=-fcommon - https://gcc.gnu.org/gcc-10/porting_to.html
+# Fixes: multiple definition of `loc_nodelay'; nettest_bsd.o:./src/nettest_bsd.c:206: first defined here
+RUN git clone https://github.com/HewlettPackard/netperf.git \
+	&& cd netperf/ \
+	&& ./autogen.sh	\
+	&& chmod a+x configure \
+	&& ./configure --prefix=/usr CFLAGS=-fcommon \
+	&& make \
+	&& make install \
+	&& cd ..
+
+RUN git clone --depth 1 --branch 3.9 https://github.com/esnet/iperf.git \
+	&& cd iperf \
+	&& ./configure --prefix=/usr \
+	&& make \
+	&& make install \
+	&& cd .. 
+
+# Cleaning
+RUN rm -rf netperf \
+	&& rm -rf iperf \
+	&& rm -f /usr/share/info/netperf.info \
+	&& strip -s /usr/bin/netperf /usr/bin/netserver \
+	&& apk del build-base texinfo git automake libtool intltool autoconf linux-headers \
+	&& rm -rf /var/cache/apk/*
```

Comment:
You want to merge all those RUN steps together, otherwise the last one is pretty much useless:
```
$ docker images | grep cilium/netperf
cilium/netperf        sctp                02a78ff8af49        20 seconds ago      319MB
cilium/netperf        1.0                 e3ac04dffddf        5 months ago        15.4MB
cilium/netperf        0.0.2               30b0ec80502c        8 months ago        16MB
```

Reply:
Yes! Thank you! Done! =]